tests/e2e/framework/framework.go

```diff
@@ -203,7 +203,7 @@ func (c *CommonConfig) RunTest(m runnable) int {
 		ret = m.Run()
 	}
 	if err := c.saveLogs(ret); err != nil {
-		glog.Warning("Log saving inconplete: %s", err)
+		glog.Warning("Log saving incomplete")
```

Comment:
IMO some amount of duplicate logging for errors doesn't hurt (vs missing logging/details), it seems odd to log something when err !=nil without logging the error (even if saveLogs does log already)